crates/starknet-server/tests/test_advancing_time.rs

```diff
@@ -72,22 +67,177 @@ mod advancing_time_tests {
             .await
             .unwrap();
 
-        let new_contract_address = get_udc_deployed_address(
+        get_udc_deployed_address(
             FieldElement::ZERO,
             declaration_result.class_hash,
             &starknet_rs_core::utils::UdcUniqueness::NotUnique,
             &[],
+        )
+    }
+
+    pub async fn get_current_timestamp(
+        devnet: &BackgroundDevnet,
+        contract_address: FieldElement,
```

Comment:
It would be clearer to call this `timestamp_contract_address`.

Reply:
renamed in 5b8dba1ad980cd3310fd4a14ed78eb636e12b8ea